lightning/src/ln/chan_utils.rs

```diff
@@ -42,6 +42,11 @@ use chain;
 use util::crypto::sign;
 
 pub(crate) const MAX_HTLCS: u16 = 483;
+pub(crate) const OFFERED_HTLC_SCRIPT_WEIGHT: usize = 133;
+pub(crate) const OFFERED_HTLC_SCRIPT_WEIGHT_ANCHORS: usize = 136;
+// The weight range for `accepted_htlc_script` encompasses both the non-anchors and anchors variants.
+pub(crate) const MIN_ACCEPTED_HTLC_SCRIPT_WEIGHT: usize = 136;
```

Comment:
nit: "The `accepted_htlc_script` size can vary in function of CLTV argument value"